mibidata/tiff.py

```diff
@@ -361,39 +371,59 @@ def _convert_from_previous(description):
         description['mibi.aperture'] = mi.MibiImage.parse_aperture(
             description['mibi.aperture'])
 
-def _get_page_data(page, description, metadata, channels):
+def _get_page_data(page, description, metadata, channels, inc_channels=None):
     """Adds to metadata and channel info for single TIFF page.
 
     Args:
         page: Single page in TIFF file.
         description: Decoded JSON description.
         metadata: Dictionary of metadata for entire TIFF file to add to.
         channels: List of channels for entire TIFF file to add to.
+        inc_channels: If set, only these channels will be added to channel
+                      parameter.
+
+    Returns:
+        include: Boolean representing whether channel should be included or not.
     """
-    channels.append((description['channel.mass'],
-                     description['channel.target']))
     # Get metadata on first SIMS page only
     if not metadata:
         metadata.update(_page_metadata(page, description))
+    if inc_channels:
```

Comment:
You could probably combine these two if statements into a single one with `and`.